packages/protocol/test-sol/TestWithUtils08.sol

```diff
@@ -139,6 +139,12 @@ contract TestWithUtils08 is ForgeTest, TestConstants, IsL2Check, PrecompilesOver
     return (addr, pk);
   }
 
+
+  function assertAlmostEqual(uint256 actual, uint256 expected, uint256 margin, string memory message) public {
```

Comment:
could this be acheived using one of the Foundry supported functions `assertApproxEqAbs`; `assertApproxEqAbsDecimal`; `assertApproxEqRel` or `assertApproxEqRelDecimal`?

ref: https://book.getfoundry.sh/reference/forge-std/assertApproxEqAbs